test/plugin/scenarios/pulsar-scenario/src/main/java/test/apache/skywalking/apm/testcase/pulsar/controller/CaseController.java

```diff
@@ -48,10 +53,25 @@
 
     @RequestMapping("/pulsar-case")
     @ResponseBody
-    public String pulsarCase() throws PulsarClientException, InterruptedException {
+    public String pulsarCase() {
 
-        String topic = "test";
+        String topicOnePartition = "test";
+        String topicMultiPartition = "testMultiPartition";
 
+        try {
+            createTopic(topicOnePartition, 1);
```

Comment:
From this change, the test only covered the partitioned topic, we can call create topic only for the partitions > 1?

Reply:
OK,I will call `doSendAndReceiveMessage` method directly for non-partitioned topic.

Reply:
done